samcli/commands/init/init_templates.py

```diff
@@ -262,3 +264,26 @@ def template_does_not_meet_filter_criteria(
         or (package_type and package_type != template.get("packageType"))
         or (dependency_manager and dependency_manager != template.get("dependencyManager"))
     )
+
+
+def filter_value_matches_template_runtime(filter_value, template_runtime):
+    """
+    Validate if the filter value matches template runtimes from the manifest file
+
+    Parameters
+    ----------
+    filter_value : str
+        Lambda runtime used to filter through data generated from the manifest
+    template_runtime : str
+        Runtime of the template in view
+
+    Returns
+    -------
+    bool
+        True if there is a match else False
+    """
+    if is_custom_runtime(filter_value) and filter_value != get_custom_runtime_base_runtime(template_runtime):
```

Comment:
`is_custom_runtime` is already calling `get_custom_runtime_base_runtime` in it, why do we need to call it here again?